### PR TITLE
Update mock

### PR DESCRIPTION
**Pull request by johnsonw**
Update mock

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -83,22 +83,18 @@ Vagrant.configure("2") do |config|
...
-    systemctl start docker
-    docker rm mock -f
+    yum install -y http://download.zfsonlinux.org/epel/zfs-release.el7_5.noarch.rpm
+    yum -y copr enable managerforlustre/manager-for-lustre-devel
     rm -rf /builddir
     cp -r /vagrant /builddir
     cd /builddir
-    npm run mock
+    cp /dev/null ~/provision-time.txt
+    date >> ~/provision-time.txt
```
Comment (jgrund) [inline Vagrantfile]: Do we want to keep this in?

File: Vagrantfile [hunk 2]
```diff
@@ -83,22 +83,18 @@ Vagrant.configure("2") do |config|
...
+    ./mock-build.sh
     PACKAGE_VERSION=$(node -p -e "require('./package.json').version")
     RPM_NAME=iml-device-scanner-$PACKAGE_VERSION-1.el7.centos.x86_64.rpm
```
Comment (jgrund) [inline Vagrantfile]: Wonder if we need to do this anymore.

Would:

`yum install -y ./iml-device-scanner-*.x86_64.rpm`

work?

Comment (johnsonw) [reply Vagrantfile]: I thought about the same thing. If you're good with it I will change it to that.

File: Vagrantfile (hunk 1 above)
Comment (johnsonw) [reply Vagrantfile]: Up to you. I added it for tacking purposes but if you want me to remove it I will.

**Comment by jgrund** [comment]
Looks like this cut off about 5 mins

**Pull request merged**